examples/hexagon_row.rs

```diff
@@ -3,7 +3,7 @@ use bevy_ecs_tilemap::prelude::*;
 mod helpers;
 
 // Side length of a colored quadrant (in "number of tiles").
-const QUADRANT_SIDE_LENGTH: u32 = 32;
+const QUADRANT_SIDE_LENGTH: u32 = 80;
```

Comment:
Not really an issue but this produces 3x3 render chunks. Probably best for testing.